### DISCUSSION TITLE
Github showing my user name from gitlab account

### DISCUSSION THREAD

**Discussion by shreekrishnaacharya** [community/community#120832] [Repositories] [answered]
### Select Topic Area

Bug

### Body

I am facing issues in github. in my laptop i have both github and gitlab account. and i am using both. But in case of github whenever i push to my repositiry. i get user name from my gitlab account ie krishna.ac18 instead of github user name. ie shreekrishnaacharya. with this i am not getting contribution mark in my profile.
I have attached the image from my public repositiry.
![Screenshot from 2024-04-23 14-37-00](https://github.com/community/community/assets/24873359/14143a82-cb1e-4f0a-a111-8523fc8f6359)

**Comment by queenofcorgis** [comment, marked as answer]
Hi @shreekrishnaacharya 👋🏼 

Have you reviewed this documentation: https://docs.github.com/en/pull-requests/committing-changes-to-your-project/troubleshooting-commits/why-are-my-commits-linked-to-the-wrong-user

I think you may find it helpful!
